Google Play Music/JSBound.cs

```diff
@@ -95,14 +95,11 @@ public void goBig()
         // Fired from javascript when a different song has been playing for over half it's duration. (As according to last.fm's scrobbling spec)
         public void songScrobbleRequest(string song, string artist, string album, int timestamp)
         {
-            try
+            // Check last.fm user authenticated before attempting to "scrobble" the current song.
+            if (LastFM.user_key != null)
             {
                 new LastFM().scrobbleTrack(artist, song, timestamp).Wait();
             }
-            catch (Exception e)
```

Comment:
Although the code does indeed look nicer like this, the try catch is there to also catch the rare case where the user has changed their password while the application is still open and in this case the `user_key` will be rejected by last.fm and and exception will be thrown.

If you can come up with a way of handling that case then that would be great